GameServerLib/Items/ItemData.cs

```diff
@@ -0,0 +1,129 @@
+using GameServerCore.Domain;
+using LeagueSandbox.GameServer.Content;
+using LeagueSandbox.GameServer.GameObjects.Stats;
+
+namespace LeagueSandbox.GameServer.Items
+{
+    public class ItemData : StatsModifier, IItemData
+    {
+        // Meta
+        public int ItemId { get; private set; }
+        public string Name { get; private set; }
+
+        // General
+        public int MaxStack { get; private set; }
+        public int Price { get; private set; }
+        public string ItemGroup { get; private set; }
+        public string SpellName { get; private set; }
+        public float SellBackModifier { get; private set; }
+
+        // Recipes
+        public int RecipeItem1 { get; private set; }
+        public int RecipeItem2 { get; private set; }
+        public int RecipeItem3 { get; private set; }
+        public int RecipeItem4 { get; private set; }
```

Comment:
do you need only 4? maybe use array? and do `RecipeItem[i] = content.GetInt("Data", "RecipeItem{i}", -1),`.

Reply:
Yep, there doesn't seem to be a good reason to limit this to just 4 items. My vote also goes out to arrays (for potential custom content in the future).